java/apps/authentication-portal/src/main/webapp/fido2-auth.jsp

```diff
@@ -174,8 +174,8 @@
         $(document).ready(function () {
             var myaccountUrl = '<%=application.getInitParameter("MyAccountURL")%>';
 
-            if ("<%=tenantDomain%>" !== "" || "<%=tenantDomain%>" !== "null") {
-                myaccountUrl = myaccountUrl + "/t/" + "<%=tenantDomain%>";
+            if ("<%=Encode.forJavaScriptBlock(tenantDomain)%>" !== "" || "<%=Encode.forJavaScriptBlock(tenantDomain)%>" !== "null") {
```

Comment:
Shall we introduce a var with encoded tenant domain and use that in the 3 mentioned places.